process/release_process.md

```diff
@@ -0,0 +1,154 @@
+
+## How to Create a Release?
+
+### xt-core
+
+- For the xt-core the evolutionary development is used in the **'main'** branch.
+- Each xt-core repository's **'main'** branch uses [semantic versioning](https://semver.org) to create release tags, such as v1.1.1, v2.2.2, etc.
+- Only repositories directly referenced by the products in the moulin configuration *.yaml files are placed inside the **'xt-core'** entity. Those repositories are:
+  - xt-common - https://github.com/xen-troops/meta-xt-common
+  - xt-rcar - https://github.com/xen-troops/meta-xt-rcar
+  - xt-rpi5 - https://github.com/xen-troops/meta-xt-rpi5
+  - xt-vhost - https://github.com/xen-troops/meta-xt-vhost
+
+  This is necessary so that products can reference these repositories by their release tags - v1.0.0, v2.1.0, etc. For example:
+
+  - type: git
+  - url: https://github.com/xen-troops/meta-xt-common.git
+  - rev: v1.0.0
+
+- All other xen-troops repositories that get into the product through the above-mentioned **xt-core** repositories will be referenced in the Bitbake recipes using the commit IDs or the release tags ( or branches ) based on community-based naming. **This means no default usage of the Bitbake 'SRCREV = "${AUTOREV}"'!**
+
+  Such repositories are:
+    
+	- backends - https://github.com/xen-troops/snd_be, https://github.com/xen-troops/displ_be, https://github.com/xen-troops/camera_be
+    - xen - https://github.com/xen-troops/xen
+    - linux - https://github.com/xen-troops/linux
+    - optee-os - https://github.com/xen-troops/optee_os, https://github.com/xen-troops/optee_client, https://github.com/xen-troops/optee_test
+    - many others.
+
+  If necessary, the commit IDs for such repositories are taken from a particular version of **xt-core**. The not **xt-core** repositories are not released without a real need and do not receive a release tag as in **xt-core**, to avoid unnecessary accumulation of tags without changes for years and to avoid duplicating community versioning. That means that:
+
+  - for **snd_be**, the commit ID should be used
+  - while for **xen** / **linux** / **optee-os**, the corresponding community versions or xen-troops branches should be used with **???-xt-???** naming.
+
+- Commits to **xt-core** are merged into the **'main'** branch based on evolutionary edits.
+- Additional branches in **xt-core** repositories can be temporarily created to meet the developer's needs.
+
+  **Finally, such branches should be merged into the main branch or closed!**
+- The tag-version of the **xt-core** entity changes only if the commit to a specific xt-core repository breaks the compilation with other xt-core repositories, i.e., the change requires edits in several xt-core repositories at once.

```

Comment:
Not only compilation. If it compiles but does not work - what sense of having it in this state? 

Ideally we want to have `min/max_version` mechanism, but it will be too much hassle.